Source/Core/Core/PowerPC/JitCommon/JitCache.h

```diff
@@ -67,7 +68,6 @@ class ValidBlockBitSet final
 	};
 	std::unique_ptr<u32[]> m_valid_block;
```

Comment:
m_valid_block isn't used anywhere outside of the class. This should remain private.